electron-app/src/filters.ts

```diff
@@ -20,6 +20,14 @@ function formatPrice(value: BigNumber, precision: number) {
   return value.toFormat(precision);
 }
 
+function capitalize(string: string): string {
+  return string[0].toUpperCase() + string.slice(1);
+}
+
+function roundDown(value: BigNumber, precision: number) {
+  return value.toFormat(precision, 1);
```

Comment:
Any idea what this 1 here is? Is that one of BigNumbers rounding schemes? if yes they should have some constants. We should use them instead.

Reply:
Yes, 1 = ROUNDDOWN as per https://mikemcl.github.io/bignumber.js/ . I will convert to the constant.